src/index.html

```diff
@@ -8,7 +8,7 @@
   	<script async defer src="//platform.instagram.com/en_US/embeds.js"></script>
 	<link rel="stylesheet" href="all.css">
 	<style>
-		ng-view {
+		/*ng-view {
```

Comment:
so, totally don't merge this since it's got the commented code in it - i'm curious if there's a way to work around this transition / omit it for transitions between portfolio pages? it's causing some weirdness with the loader between portfolio pages, specifically it's showing the loader fading in twice. you can try checking out this branch and uncommenting this block to see what i mean, it's apparent on portfolios that take 1s+ to load. maybe there's a way to work with this transition that i'm missing? would love to know your thoughts!

Reply:
we are getting rid of ng-view transitions, so no need to worry about that, I will remove

Reply:
Groovy, gonna delete merge and deploy then, keep me posted if you need anything from moi